Podfile

```diff
@@ -13,7 +13,7 @@ use_frameworks!
 def shared_pods
   
   # CruGlobal pods
-  pod 'GodToolsShared', '1.0.1'
+  pod 'GodToolsShared', '1.1.0-SNAPSHOT'
```

Comment:
@levieggertcru let me know if you are close to releasing a build with this PR and I'll cut a release version of the GodToolsShared library

Reply:
I might back port the small change made for this to a 1.0.2 release. There some changes I'm needing to make to add support for the new cyoa page collection page type.